### DISCUSSION TITLE
superiorOs Thirteen sync error need clean build

### DISCUSSION THREAD

**Discussion by ahmedhridoy371** [ROM-builders/temporary#19305] [General]
i guess superiorOs-thirteen need to clean because facing sync error also source are good no source side problem (tested) on personal server but in cirrus giving sync error.

here some build reference:
https://cirrus-ci.com/build/4747498120544256
https://cirrus-ci.com/task/5601582432124928

Also Sorry For my bad english

**Comment by ahmedhridoy371** [comment]
@Apon77

**Reply by Apon77** [reply]
I get notifications without tag. No need to tag explicitly. Also none of those are your builds

**Reply by ahmedhridoy371** [reply]
https://cirrus-ci.com/task/6036139639832576
